CHANGE_LOG.md

```diff
@@ -68,4 +68,5 @@ All notable changes to this project will be documented in this file.## [Unreleas
 - Removed isReconnectEnabled from props
 - Fixed OutOfOffice Chat Button Icon appearance
 - Adding support for customizing widget scroll bar
-- Clearing live chat context for raise error event
\ No newline at end of file
+- Clearing live chat context for raise error event
+- Refactoring input validation pane
```

Comment:
Please add the Key up and Key down issue fix in ChangeLog

Reply:
Added